### PR TITLE
I repogram boyfreind

### PR DESCRIPTION
**Pull request by TheKillerA1**
Just doing this to pass time before “the wave”

### CODE REVIEW

**Comment by MemeHoovy** [comment]
no, fucking no

**Comment by LilDrippyMyFnf** [comment]
no,hell no

File: source/Boyfriend.hx [hunk 1]
```diff
@@ -25,19 +25,19 @@ class Boyfriend extends Character
...
+			if (animation.curAnim.name == 'kill' && animation.curAnim.finished && startedDeath)
 			{
-				playAnim('deathLoop');
+				playAnim('die');
 			}
 		}
 
-		super.update(elapsed);
+		ultra.update(elapsed);
 	}
```
Comment (LilDrippyMyFnf) [inline source/Boyfriend.hx:42]: why the hell did you do this?

**Review by MemeHoovy — CHANGES_REQUESTED** [review]
go fuck yourself

**Comment by LilDrippyMyFnf** [comment]
> go fuck yourself you piece of garbage

bruh

**Comment by LilDrippyMyFnf** [comment]
Screenshot 2022-10-09 11 50 22 PM

**Comment by TheKillerA1** [comment]
> go fuck yourself you piece of garbage

Live adaptation of “bozo the clown”

**Comment by MemeHoovy** [comment]
> > go fuck yourself you piece of garbage
> 
> Live adaptation of “bozo the clown”

your a fucking idiot

**Comment by LilDrippyMyFnf** [comment]
> > go fuck yourself you piece of garbage
> 
> Live adaptation of “bozo the clown”

like you can roast,dity ass

**Comment by TheKillerA1** [comment]
> > > go fuck yourself you piece of garbage
> > 
> > 
> > Live adaptation of “bozo the clown”
> 
> your a fucking idiot

Man you seriously cannot see the layer of irony this pr has, I mean. Hate to break the trolling but you literally look like that 44 year old man watching hentai every 5 seconds

**Comment by LilDrippyMyFnf** [comment]
> > > > go fuck yourself you piece of garbage
> > > 
> > > 
> > > Live adaptation of “bozo the clown”
> > 
> > 
> > your a fucking idiot
> 
> Man you seriously cannot see the layer of irony this pr has, I mean. Hate to break the trolling but you literally look like that 44 year old man watching hentai every 5 seconds

sure with yo old ass eye

**Comment by LilDrippyMyFnf** [comment]
> > > > > go fuck yourself you piece of garbage
> > > > 
> > > > 
> > > > Live adaptation of “bozo the clown”
> > > 
> > > 
> > > your a fucking idiot
> > 
> > 
> > Man you seriously cannot see the layer of irony this pr has, I mean. Hate to break the trolling but you literally look like that 44 year old man watching hentai every 5 seconds
> 
> sure with yo old ass eye

at least i can draw a decent pfp BITCH

**Comment by TheKillerA1** [comment]
> > > > > > go fuck yourself you piece of garbage
> > > > > 
> > > > > 
> > > > > Live adaptation of “bozo the clown”
> > > > 
> > > > 
> > > > your a fucking idiot
> > > 
> > > 
> > > Man you seriously cannot see the layer of irony this pr has, I mean. Hate to break the trolling but you literally look like that 44 year old man watching hentai every 5 seconds
> > 
> > 
> > sure with yo old ass eye
> 
> at least i can draw a decent pfp BITCH

5218EA31-C31B-4E19-917E-4BDD5E124C0B
Hehe hehe funni amung us sus

**Comment by LilDrippyMyFnf** [comment]
> > > > > > > go fuck yourself you piece of garbage
> > > > > > 
> > > > > > 
> > > > > > Live adaptation of “bozo the clown”
> > > > > 
> > > > > 
> > > > > your a fucking idiot
> > > > 
> > > > 
> > > > Man you seriously cannot see the layer of irony this pr has, I mean. Hate to break the trolling but you literally look like that 44 year old man watching hentai every 5 seconds
> > > 
> > > 
> > > sure with yo old ass eye
> > 
> > 
> > at least i can draw a decent pfp BITCH
> 
> 5218EA31-C31B-4E19-917E-4BDD5E124C0B [ 5218EA31-C31B-4E19-917E-4BDD5E124C0B ](https://user-images.githubusercontent.com/115441433/194797410-672d312d-f09e-4d9e-b0a1-b027d22beed7.gif) [ ](https://user-images.githubusercontent.com/115441433/194797410-672d312d-f09e-4d9e-b0a1-b027d22beed7.gif) Hehe hehe funni amung us sus

witch yo dumbass eye

**Comment by LilDrippyMyFnf** [comment]
i beat if i save it,its a JPEG CUZ YOU ON A PHONE

**Comment by TheKillerA1** [comment]
> i beat if i save it,its a JPEG CUZ YOU ON A PHONE

I think it’s that you are the one on the phone, little timmy.

**Comment by ShadowMario** [comment]
> Just doing this to pass time before “the wave”

waving my nuts ain't ya

**Pull request closed without merging**